### ISSUE TITLE
Credentials are not saved

### ISSUE THREAD

**Issue by alhafoudh** [bladerunner2020/node-red-contrib-mikrotik#3]
When I configure the node, it works, but credentials are not persisted to flow and node-red keeps crashing on start with error message:

```
21 Nov 18:39:51 - TypeError: Cannot read property 'length' of undefined
    at Channel.loginDone (/Users/xxx/node-red-sample/node_modules/mikronode-ng/lib/connection.js:587:49)
```

**Comment by alhafoudh** [comment]
Fixed in PR #4

**Comment by bladerunner2020** [comment]
I accepted PR and added some additional changes.
